src/qt/locale/bitcoin_ar.ts

```diff
@@ -3693,891 +3624,891 @@ Use this functionality with extreme caution.</source>
     </message>
     <message>
         <source>Keypool ran out, please call keypoolrefill first</source>
-        <translation type="unfinished"></translation>
+        <translation>نفد Keypool، يرجى الاتصال بـ keypoolrefill أولاً</translation>
     </message>
     <message>
         <source>Listen for JSON-RPC connections on &lt;port&gt; (default: %u or testnet: %u)</source>
-        <translation type="unfinished"></translation>
+        <translation>استمع لاتصالات JSON-RPC على &lt;port&gt; (الافتراضي: %u أو شبكة الاختبار: %u)</translation>
     </message>
     <message>
         <source>Listen for connections on &lt;port&gt; (default: %u or testnet: %u)</source>
-        <translation type="unfinished"></translation>
+        <translation>استمع للاتصالات على &lt;port&gt; (الافتراضي: %u أو شبكة الاختبار: %u)</translation>
     </message>
     <message>
         <source>Loading banlist...</source>
-        <translation type="unfinished"></translation>
+        <translation>جارٍ تحميل قائمة الحظر</translation>
     </message>
     <message>
         <source>Location of the auth cookie (default: data dir)</source>
-        <translation type="unfinished"></translation>
+        <translation>موقع ملف تعريف ارتباط المصادقة (الافتراضي: data dir)</translation>
     </message>
     <message>
         <source>Maintain at most &lt;n&gt; connections to peers (default: %u)</source>
-        <translation type="unfinished"></translation>
+        <translation>الحفاظ على الأكثر &lt;n&gt; الاتصالات بالأقران (الافتراضي: %u)</translation>
     </message>
     <message>
         <source>Many generous</source>
-        <translation type="unfinished"></translation>
+        <translation>كثيرة سخية</translation>
     </message>
     <message>
         <source>Maximum per-connection receive buffer, &lt;n&gt;*1000 bytes (default: %u)</source>
-        <translation type="unfinished"></translation>
+        <translation>الحد الأقصى لمخزن الاستقبال المؤقت لكل اتصال، &lt;n&gt;*1000 بايت (الافتراضي: %u)</translation>
     </message>
     <message>
         <source>Maximum per-connection send buffer, &lt;n&gt;*1000 bytes (default: %u)</source>
-        <translation type="unfinished"></translation>
+        <translation>الحد الأقصى لمخزن الإرسال المؤقت لكل اتصال، &lt;n&gt;*1000 بايت (الافتراضي: %u)</translation>
     </message>
     <message>
         <source>Minimum</source>
-        <translation type="unfinished"></translation>
+        <translation>الحد الأدنى</translation>
     </message>
     <message>
         <source>More</source>
-        <translation type="unfinished"></translation>
+        <translation>أكثر</translation>
     </message>
     <message>
         <source>Need to specify a port with -whitebind: &apos;%s&apos;</source>
-        <translation type="unfinished"></translation>
+        <translation>تحتاج إلى تحديد منفذ باستخدام -whitebind: &apos;%&apos;</translation>
```

Comment:
should be `&apos;%s&apos;`

note the extra `s`

apologies for the many comments, but the linter only catches one error at a time

Reply:
> the linter only catches one error at a time

only one XML error at a time because that prevents from parsing further. Semantic errors like `%s` and `%1` mismatches are all checked at once.

Reply:
Sorry for the such errors 🥸

Reply:
No probs. I just have to remember to approve the tests. So the sorry for such slow is all mine.